inst/extdata/energy/A25.globaltech_co2capture.csv

```diff
@@ -0,0 +1 @@
+# File: A25.globaltech_co2capture.csv
# Title: Hydrogen technology capture fractions
# Units: Unitless
supplysector,subsector,technology,1971,2020,2035,2050,2065,2100
H2 central production,coal,coal chemical CCS,0.91,0.91,0.92,0.93,0.94,0.94
H2 central production,gas,natural gas steam reforming CCS,0.91,0.91,0.92,0.93,0.94,0.94
H2 central production,biomass,biomass to H2 CCS,0.91,0.91,0.92,0.93,0.94,0.94

```

Comment:
Line endings messed up (windows, not *nix). I can fix, or re-save in a good text editor.